src/fides/data/sample_project/privacy_center/config/config.json

```diff
@@ -9,10 +9,10 @@
       "policy_key": "default_access_policy",
       "icon_path": "/assets/access.svg",
       "title": "Access your data",
-      "description": "We will provide you a report of all your personal data.",
+      "description": "Please provide either an email or a phone number. We will provide you a report of all your personal data.",
       "identity_inputs": {
         "name": "optional",
-        "email": "required",
+        "email": "optional",
```

Comment:
NOTE: there's a third config.json that should be updated, which is src/fides/data/test_env/privacy_center/config/config.json.

This `test_env` config is the one used for our main regression testing, so it should be updated as well to reflect this new default behaviour